### PR TITLE
Add presence for discordextremelist.xyz!

### PR DESCRIPTION
**Pull request by Cairo2k18**
**Some Examples:**
image
image
image
image
image
image
image

### CODE REVIEW

File: websites/D/Discord Extreme List/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    "url": [
+      "discordextremelist.xyz",
+      "staging.discordextremelist.xyz"
+    ],
+    "description": {
+      "en": "Discord's unbiased list, giving bots and servers of all sizes a big chance!",
+      "es": "¡Lista imparcial de Discord, que ofrece a los bots y servidores de todos los tamaños una gran oportunidad!",
+      "pt": "Lista de discord imparcial, dá a bots e servidores de todos os tamanhos uma grande chance!",
+      "he": "הרשימה הבלתי משוחדת של דיסקורד, נותנת לבוטים ושרתים בכל הגדלים סיכוי גדול!",
+      "hu": "Discord-nak elfogulatlan listája, megadja a botoknak és a szervereknek a nagy lehetőséget!",
```
Comment (Bas950) [inline websites/D/Discord Extreme List/dist/metadata.json:15]: he_IL

Comment (Bas950) [reply websites/D/Discord Extreme List/dist/metadata.json:15]: Double check the others as well: https://api.premid.app/v2/langFile/list

File: websites/D/Discord Extreme List/tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "../../../tsconfig.json",
+    "compilerOptions": {
+      "outDir": "./dist/"
+    }
+}
+
```
Comment (Bas950) [inline websites/D/Discord Extreme List/tsconfig.json:7]: Remove that extra line xD

File: websites/D/Discord Extreme List/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      "de": "Discord's unbefangene Liste, gibt Bots und Server aller grössen eine grosse Chance!",
+      "nl": "Discord's onbevooroordeelde lijst, die bots en servers van elke omvang een grote kans geeft!",
+      "sv_SE": "Discords opartiska lista, ger alla bots och servrar en stor chans!",
+      "tr": "Discord'un tarafsız Bot Listesi , Botunuz ve sizin için oldukça büyük bir şans!"
+    },
+    "service": "Discord Extreme List",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/tMrrgGu.png",
+    "thumbnail": "https://i.imgur.com/ewofSNN.png",
```
Comment (frozaum) [inline websites/D/Discord Extreme List/dist/metadata.json:24]: image

Comment (Cairo2k18) [reply websites/D/Discord Extreme List/dist/metadata.json:24]: LMAO

**Pull request merged**